src/views/index.html

```diff
@@ -1,5 +1,9 @@
 {% extends 'base.html' %}
 {% block content %}
+
+{# MLH Badge #}
+<a id="mlh-trust-badge" style="display:block;max-width:100px;min-width:70px;position:fixed;right:20px;top:0;width:10%;z-index:10000" href="https://mlh.io/seasons/eu-2017/events?utm_source=eu-2017&utm_medium=TrustBadge&utm_campaign=eu-2017&utm_content=gray" target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2017/gray.svg" alt="Major League Hacking 2017 Hackathon Season" style="width:100%"></a>
```

Comment:
We had issues last year with these default styles last year:

- The badge had a habit of overlaying our title text (this may be different now)
- Being `position: fixed` gets annoying fast
- Inline styles aren't great for maintainability